src/PAModel/Utility/FilePath.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.PowerPlatform.Formulas.Tools.Utility
 {
     internal class FilePath
     {
+        private const int MAX_PATH = 260;
```

Comment:
This is the max path for windows in general, but what if we're unpacking to a directory that already has a path that's, say, 100 char long? I'd suggest that we consider having a shorter max path on our end so we're not pushing the absolute limit.

Reply:
this is to test the length of the absolute path

Reply:
There's some very good discussion on Stackoverflow around here, [https://stackoverflow.com/questions/23588944/better-to-check-if-length-exceeds-max-path-or-catch-pathtoolongexception?noredirect=1&lq=1](https://stackoverflow.com/questions/23588944/better-to-check-if-length-exceeds-max-path-or-catch-pathtoolongexception?noredirect=1&lq=1)